src/Frost/Effects/Stack.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Frost.Effects.Stack where
+
+import FrostError
+import Frost.Effects.Sys
+
+import Polysemy
+import Polysemy.Error
+
+type SpecName = String
+type TestName = String
+
+data Stack m a where
+  Clean :: Stack m String
+  Build :: Stack m String
+  Exec :: String -> Stack m String
+  Test :: Stack m String
+  TestMatch :: SpecName -> TestName -> Stack m String
+
+makeSem ''Stack
+
+runStack :: Member Sys r => Sem (Stack ': r) a -> Sem r a
+runStack = interpret $ \case
+  Clean                       -> stack "clean"
+  Build                       -> stack "build"
+  Exec what                   -> stack $ "exec " ++ what
+  Test                        -> stack "test"
+  TestMatch specName testName -> stack $ "test --match " ++ (show $ matchStr specName testName)
+  where
+    stack :: Member Sys r => String -> Sem r String
+    stack arg = fmap snd (cmd $ "stack --no-terminal " ++ arg)
+    matchStr specName testName = "/Stack/" ++ specName ++ "/" ++ testName ++ "/"
```

Comment:
Why the `/Stack` prefix?

Reply:
stack used to add `/Stack`, now it does not 🤷‍♂

Reply:
so it means that for now we only support latest stack